src/app/pages/Market/crypto-exchange/steps/OrderCreation/components/InfoCard.tsx

```diff
@@ -1,18 +1,33 @@
 import React, { memo } from 'react';
 
+import Money from 'app/atoms/Money';
 import { T } from 'lib/i18n';
 
 import { InfoContainer, InfoRaw } from '../../../components/InfoBlock';
-import { EXOLIX_PRIVICY_LINK, EXOLIX_TERMS_LINK } from '../../../config';
+import { EXOLIX_PRIVICY_LINK, EXOLIX_TERMS_LINK, VALUE_PLACEHOLDER } from '../../../config';
 
 interface Props {
-  exchangeRate: string;
+  rate: number | nullish;
+  inputCurrencyCode: string;
+  outputCurrencyCode: string;
 }
 
-export const InfoCard = memo<Props>(({ exchangeRate }) => (
+export const InfoCard = memo<Props>(({ rate, inputCurrencyCode, outputCurrencyCode }) => (
   <InfoContainer className="mb-8">
     <InfoRaw bottomSeparator title="exchangeRate">
-      <span className="p-1 text-font-description">{exchangeRate}</span>
+      <span className="p-1 text-font-description">
+        {rate ? (
+          <>
+            <span>{`1 ${inputCurrencyCode} ≈ `}</span>
+            <Money cryptoDecimals={2} smallFractionFont={false} tooltipPlacement="bottom">
```

Comment:
```suggestion
            <Money smallFractionFont={false} tooltipPlacement="bottom">
```
See https://madfishgroup.slack.com/archives/C0535E9N3PE/p1734347773742049?thread_ts=1734346387.881139&cid=C0535E9N3PE for reason

Reply:
there could be number like `5555555555.11111111` which will look weird in ui, so added a bit extra logic